scipy/stats/_continuous_distns.py

```diff
@@ -6123,6 +6141,24 @@ def _stats(self, b, moments='mv'):
     def _entropy(self, c):
         return 1 + 1.0/c - np.log(c)
 
+    def fit(self, data, *args, **kwds):
+        parameters = _check_fit_input_parameters(self, data, args, kwds)
+        data, fshape, floc, fscale = parameters
+        if floc is None:
+            return super(pareto_gen, self).fit(data, **kwds)
+        if np.any(data - floc < (fscale if fscale else 0)):
+            raise FitDataError("pareto", lower=0, upper=np.inf)
```

Comment:
The lower bound of the support for the standardized values is 1 (i.e. we require `(data - loc)/scale > 1`).
```suggestion
            raise FitDataError("pareto", lower=1, upper=np.inf)
```

Reply:
Nice catch. That was leftover from when the message was the lower bound of just `x`